LabExT/View/StageCalibration/StageCalibrationView.py

```diff
@@ -0,0 +1,197 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+"""
+LabExT  Copyright (C) 2022  ETH Zurich and Polariton Technologies AG
+This program is free software and comes with ABSOLUTELY NO WARRANTY; for details see LICENSE file.
+"""
+
+from tkinter import W, Frame, Label, OptionMenu, StringVar, Button, messagebox, NORMAL, DISABLED, LEFT, RIGHT, TOP, X
+from functools import partial
+from itertools import product
+from bidict import bidict
+from typing import Type
+
+from LabExT.Movement.Calibration import AxesRotation, Calibration, Axis, CalibrationError, Direction
+from LabExT.Utils import run_with_wait_window
+from LabExT.View.Controls.CustomFrame import CustomFrame
+from LabExT.View.Controls.Wizard import Wizard
+
+
+class StageCalibrationView(Wizard):
+
+    STAGE_AXIS_OPTIONS = bidict({o: " ".join(map(str, o))
+                                for o in product(Direction, Axis)})
+
+    def __init__(self, parent, experiment_manager, controller, mover) -> None:
+        super().__init__(
+            parent,
+            width=1000,
+            height=700,
+            cancel_button_label="Cancel and Close",
+            finish_button_label="Finish and Save"
+        )
+        self.title("Stage Calibration Wizard")
+
+        self.controller = controller
+        self.experiment_manager = experiment_manager
+        self.mover = mover
+
+        # -- 1. STEP: FIX COORDINATE SYSTEM --
+        self.fix_coordinate_system_step = self.add_step(
+            self._fix_coordinate_system_step_builder,
+            title="Fix Coordinate System",
+            on_reload=self._check_axis_calibration,
+            on_next=self._on_save_axes_rotations)
+        # Step state and variables
+        self._performing_wiggle = False
+        self._current_axes_rotations = {}
+        self._axis_calibration_vars = {}
+        self._axis_wiggle_buttons = {}
+
+        for calibration in self.mover.calibrations.values():
+            self._current_axes_rotations[calibration] = AxesRotation()
+
+            axes_vars = {}
+            for chip_axis in Axis:
+                axes_vars[chip_axis] = StringVar(
+                    self.parent, self.STAGE_AXIS_OPTIONS[(Direction.POSITIVE, chip_axis)])
+                axes_vars[chip_axis].trace(
+                    W,
+                    lambda *_,
```

Comment:
What is this kind of fancy syntax? 😄

Reply:
The trace method for variables calls the method with arguments that I don't need. `*_` is like `*args`, except that the `_` implies that the argument is not used.